.github/workflows/coverage.yml

```diff
@@ -0,0 +1,63 @@
+name: coverage
```

Comment:
Shouldn't we run this job only when the CI passes? I know it's currently broken, but it might be good to run the coverage tests only when tests work?

Reply:
AFAIK, it's not possible to have workflows depend on each other (and as our CI isn't running with GitHub actions it's not even a workflow). If you know of a way to achieve it, it sounds like a good idea (but as you say, not until our CI is passing consistently).

Reply:
This GitHub action seems to give the possibility to wait for an action to execute another: https://github.com/marketplace/actions/wait-on-check. Maybe, you can leave it like this for now, and, we see when the CI is fixed?

Reply:
Good find. Let's do that :+1: